### PR TITLE
Add accept terms checkbox

### PR DESCRIPTION
**Pull request by cb372**
Add a checkbox for developers to signal acceptance of the Open Platform terms and conditions.

The "Register" button is disabled unless the box is checked.

### CODE REVIEW

File: assets/css/access.css [hunk 1]
```diff
@@ -66,7 +66,7 @@
     font-family: 'Source Sans Pro',sans-serif;
 }
 
-.button:hover {
+.button:hover:enabled {
```
Comment (cb372) [inline assets/css/access.css:69]: IE 8 and older doesn't support the `:enabled` pseudo-class, so the button will look clickable even when it's disabled. Is this acceptable? If not, I'll try and find a workaround.

Comment (gidsg) [inline assets/css/access.css:69]: I would say that is acceptable, we only fully support >= IE9 on theguardian.com so I think the same standard is OK on open platform site.

**Comment by chriswilk** [comment]
:+1: Looks good to me.  This is plenty for what we need at this time.

File: access/index.html [hunk 2]
```diff
@@ -58,17 +58,12 @@
...
 
        <div id="developer">
-                  <p>
-                    <a class="button" href="http://guardian.mashery.com/member/register">Register
-                    </a>
-                  </p>
-
                   <p class="head" style="text-align: center">
-                  By applying to register you are deemed to have accepted the <a class="link" href="http://www.theguardian.com/open-platform/terms-and-conditions">Guardian Open Platform terms and conditions</a><br />
-                  <a class="link" href="https://secure.mashery.com/login/guardian.mashery.com/">Already have a Mashery account?</a>
```
Comment (mchv) [inline access/index.html:68]:
``` html
 <a class="link" href="https://secure.mashery.com/login/guardian.mashery.com/">Already have a Mashery account?</a>
```

 May stay as it was an alternative link for users already having a mashery account, but I am not sure if it is really useful :wink:

File: assets/css/access.css (hunk 1 above)
Comment (chriswilk) [inline assets/css/access.css:69]: This is acceptable for now. :+1:

Comment (mchv) [inline assets/css/access.css:69]: I agree it is acceptable.

**Comment by mchv** [comment]
:+1:

File: access/index.html (hunk 2 above)
Comment (gidsg) [inline access/index.html:68]: I think it is fine to remove as it is an edge case that a user already has an account and if they do they can click 'Sign in' on the Mashery registration page.

**Comment by gidsg** [comment]
@cb372 as a tip for future PRs, if you are making visual changes it is helpful to include a screenshot or gif video of interaction as a quick reference for people.

**Comment by cb372** [comment]
@gidsg no problem, will do so in future!

**Comment by cb372** [comment]
For the record, here's the new interaction

untitled

**Comment by gidsg** [comment]
@cb372 feel free to Merge! :ship: (On Content API team developers are responsible for merging, deployment and checking on PROD)

**Comment by cb372** [comment]
Checked on prod, seems to be working fine.

**Pull request merged**